src/cli/llm/infer_type_name.rs

```diff
@@ -102,47 +100,27 @@ impl InferTypeName {
                     .collect(),
             };
 
-            let mut delay = 3;
-            loop {
-                let answer = wizard.ask(question.clone()).await;
-                match answer {
-                    Ok(answer) => {
-                        let name = &answer.suggestions.join(", ");
-                        for name in answer.suggestions {
-                            if config.types.contains_key(&name)
-                                || new_name_mappings.contains_key(&name)
-                            {
-                                continue;
-                            }
-                            new_name_mappings.insert(name, type_name.to_owned());
-                            break;
+            match wizard.ask(question).await {
```

Comment:
```suggestion
            let answer = wizard.ask(question).await?;
```

No need to pattern match here.